### PR TITLE
12.0 mig coop inventory

### PR DESCRIPTION
**Pull request by krunal267**
Task URL:

https://www.druidoo.io/web#id=1548&action=367&model=project.task&view_type=form&menu_id=237

### CODE REVIEW

File: purchase_package_qty/__manifest__.py [hunk 1]
```diff
@@ -40,7 +40,7 @@
         'views/purchase_order_view.xml',
         'views/stock_picking_view.xml',
         'views/account_invoice_view.xml',
-        'views/report_stockinventory.xml',
+        # 'views/report_stockinventory.xml',
```
Comment (ivantodorovich) [inline purchase_package_qty/__manifest__.py]: you're removing this view.

But also, why are we changing `purchase_package_qty`? that's not in the scope of this PR and that module is supposed to be working :thinking:

**Review by ivantodorovich — CHANGES_REQUESTED** [review]

**Comment by krunal267** [comment]
Hi @ivantodorovich 

Please check the Travis, somehow it's failing due to purchase_package_qty module

**Comment by krunal267** [comment]
Should I do the changes in the same branch or create a new branch for the change in puchase_package_qty?

**Comment by krunal267** [comment]
Hi @ivantodorovich 

The branch is ready to merge :)

**Pull request merged**